docs/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.6.2-stretch
```

Comment:
Ah, nice idea to add `Dockerfile` 👍

Reply:
BTW, I'm guessing the issues you had were due to Ruby versions, which requires messing with `rbenv` (or similar)... In these cases, a known working `Dockerfile` can be a lot nicer.

Reply:
FWIW, the issues I had was actually not the ruby versions (I think I had `rbenv` set up correctly), but the dependency on a system library (`libv8` which requires `v8` on your system) and getting the compiler to work. Here are the issues I ran into:

- The build script uses `python` and requires python2 which was annoying because I had `pyenv` hard coded to python 3.
- I needed `v8` to already have been installed since it is dynamically linked to the ruby library. `brew install v8` solved the issue.
- I could not get `libv8` to compile, because it kept running into issues with linking to the stdlib. This appears to be a known issue with using `clang++` instead of `gcc`. The workaround is to install `gcc`, but I did not want to do that as that would mess other things up on my mac. I fiddled with the flags to see if there was a workaround but ultimately gave up and switched to docker.

Reply:
Roger.